getting_started/scripting/c_sharp/c_sharp_basics.rst

```diff
@@ -135,33 +168,45 @@ The C# API uses ``PascalCase`` instead of ``snake_case`` in GDScript/C++.
 Where possible, fields and getters/setters have been converted to properties.
 In general, the C# Godot API strives to be as idiomatic as is reasonably possible.
 
-For more, see the :ref:`doc_c_sharp_differences` page.
+For more information, see the :ref:`doc_c_sharp_differences` page.
 
 Current gotchas and known issues
 --------------------------------
 
-As C# support is quite new to Godot, there are some growing pains and things that still need to be ironed out.
-Below is a list of the most important issues you should be aware of when diving into C# in Godot, but if in doubt also take a look over the official `issue tracker for Mono issues <https://github.com/godotengine/godot/labels/topic%3Amono>`_.
-
-- As explained above, the C# project isn't always kept in sync automatically when things are deleted, renamed or moved in Godot (`#12917 <https://github.com/godotengine/godot/issues/12917>`_)
-- Writing editor plugins and tool scripts in C# are not yet supported
-- Exporting Mono projects is only supported for desktop platforms (Linux, Windows and macOS). HTML5, Android, iOS and UWP are not currently supported (`#18364 comment <https://github.com/godotengine/godot/issues/18364#issuecomment-406222102>`_)
-- Attached C# scripts should refer to a class that has a class name that matches the file name and is unique within the project assembly (`#7402 comment <https://github.com/godotengine/godot/issues/7402#issuecomment-269910926>`_)
+As C# support is quite new in Godot, there are some growing pains and things
+that need to be ironed out. Below is a list of the most important issues
+you should be aware of when diving into C# in Godot, but if in doubt, also
+take a look over the official
+`issue tracker for Mono issues <https://github.com/godotengine/godot/labels/topic%3Amono>`_.
+
+- As explained above, the C# project isn't always kept in sync automatically
+  when things are deleted, renamed or moved in Godot
+  (`#12917 <https://github.com/godotengine/godot/issues/12917>`_).
+- Writing editor plugins and tool scripts in C# is not yet supported.
+- Exporting Mono projects is only supported for desktop platforms
+  (Linux, Windows and macOS). HTML5, Android, iOS and UWP are not currently supported
+  (`#18364 comment <https://github.com/godotengine/godot/issues/18364#issuecomment-406222102>`_).
+- Attached C# scripts should refer to a class that has a class name
+  that matches the file name and is unique within the project assembly
+  (`#7402 comment <https://github.com/godotengine/godot/issues/7402#issuecomment-269910926>`_).
```

Comment:
The linked issue doesn't make much sense here and my lead to confusion.
About the "unique" thing, I guess this was related to Godot not supporting namespaces back then, which is no longer the case so I would remove that.